TACC/TACC-Docs
# feat: index page

## Overview

- Add sample content to index page.
- Load Core-Styles patterns for index page.

## Related

None.

## Changes

- **adds** candidate styles for `core-styles.docs.css`
- **removes** `section` class, via JavaScript, on index page, from content wrapper
- **adds** sample content for index page
- **adds** support for [markdown blocks](https://facelessuser.github.io/pymdown-extensions/extensions/blocks/) (and uses [the `html` one](https://facelessuser.github.io/pymdown-extensions/extensions/blocks/plugins/html/))

## Testing

1. Load index page.
2. Verify cards in muted section are clickable.
3. Verify linked cards have user feedback on click, hover, focus.
4. Verify linked cards navigate accurately.
5. Verify cards that link to external site open in new tab.
6. Verify muted section has top and bottom border.
7. Verify layout generally[^1] matches [design](https://xd.adobe.com/view/dabaec5a-cc31-4c5c-876e-56d6d8cc359e-8fe9/screen/62c317eb-f68a-4fd1-8e9b-aa0e7b15e06c/specs/).

[^1]: No need to be pixel perfect. The design is rough. Fonts are still not loaded on docs.

## UI

### Sample (Based on Design)

This page was moved to `/index-sample/` after 7b7d2e1.

![docs index page](https://user-images.githubusercontent.com/62723358/223558780-365c4f4f-2779-48d0-8ee0-8da8f9125362.png)

### Index (Based on `main`)

![docs index page from main](https://user-images.githubusercontent.com/62723358/223562412-a28ad312-b571-4fbd-a665-72727be1cc9e.png)
